### PR TITLE
Skip coming soon items from featured section

### PR DESCRIPTION
**Pull request by eliezerben**

### CODE REVIEW

File: lib/components/featured_section.dart [hunk 1]
```diff
@@ -27,7 +25,10 @@ class FeaturedSection extends StatelessWidget {
     return LayoutBuilder(builder: (context, constraints) {
       const marginX = 2.0;
       final viewportFraction = (constraints.maxWidth - (32 - 2 * marginX)) / max(1, constraints.maxWidth);
-      final sectionItems = data.items.items;
+      final sectionItems = data.items.items.where((e) {
+        final episode = e.item.asOrNull<Fragment$ItemSectionItem$item$$Episode>();
+        return episode == null ? true : !isComingSoon(episode.publishDate);
```
Comment (andreasgan) [inline lib/components/featured_section.dart:30]:
```suggestion
        return episode == null || !locked && !isComingSoon(episode.publishDate);
```

**Review by andreasgan — CHANGES_REQUESTED** [review]
Wouldn't this be better?

File: lib/components/featured_section.dart (hunk 1 above)
Comment (eliezerben) [reply lib/components/featured_section.dart:30]: I guess this update will not be needed after merging this https://github.com/bcc-code/brunstadtv-app/pull/80

Comment (andreasgan) [reply lib/components/featured_section.dart:30]: No because this is more about removing "unclickable" items from featureds

Comment (andreasgan) [reply lib/components/featured_section.dart:30]: So "coming soon" was wrong, sorry, I should have said "locked"

Comment (eliezerben) [reply lib/components/featured_section.dart]: Right

**Review by andreasgan — APPROVED** [review]

**Pull request merged**